src/modules/io/Elsa.IO.Compression/Elsa.IO.Compression.csproj

```diff
@@ -12,13 +12,14 @@
         <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
         <PackageReference Include="Microsoft.Extensions.Options" />
     </ItemGroup>
-
-    <ItemGroup>
-        <ProjectReference Include="..\Elsa.IO\Elsa.IO.csproj" />
-    </ItemGroup>
-
+    
     <ItemGroup Label="Elsa" Condition="'$(UseProjectReferences)' != 'true'">
         <PackageReference Include="Elsa.Workflows.Management" />
+        <PackageReference Include="Elsa.IO" />
     </ItemGroup>
 
+    <ItemGroup Label="Elsa" Condition="'$(UseProjectReferences)' == 'true'">
+        <ProjectReference Include="..\..\..\..\..\elsa-core\src\modules\Elsa.Workflows.Management\Elsa.Workflows.Management.csproj" />
+        <ProjectReference Include="..\Elsa.IO\Elsa.IO.csproj" />
```

Comment:
This project reference should not be conditionally included; it should always be included.